comictaggerlib/main.py

```diff
@@ -206,8 +206,8 @@ def main(self) -> None:
             if self.config_load_success:
                 settings_path = self.config[0].Runtime_Options_config.user_config_dir / "settings.json"
                 if self.config_load_success:
-                    self.manager.save_file(self.config[0], settings_path)
-                print("Key set")  # noqa: T201
+                    save_file(self.config, settings_path)
```

Comment:
This needs to be renamed either use the full import `ctsettings.save_file` or rename this to save_settings or a similar name